client/signup/steps/creds-permission/index.jsx

```diff
@@ -66,6 +66,11 @@ class CredsPermissionStep extends Component {
 							"Jetpack access to your host's server to perform backups?"
 					) }
 				</p>
+				<p className="creds-permission__description">
+					{ translate(
+						'By adding credentials, you are providing us with access to your server to perform automatic actions (such as backing up or restoring your site), manually access your site in case of an emergency, and troubleshoot your support requests.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'By adding credentials, you are providing us with access to your server to perform automatic actions (such as backing up or restoring your site) or to manually access your site in case of an emergency.' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jetpack-backup-credentials-flow)